scripts/travis/release/docker-tag.sh

```diff
@@ -8,7 +8,7 @@ if [[ $TRAVIS_BRANCH =~ ^master(-patch.*)?$ ]]; then
 else
     if [[ "${TRAVIS_PULL_REQUEST_BRANCH}" != "" ]];
     then
-        export TAGS=""$TRAVIS_PULL_REQUEST_BRANCH-$TRAVIS_BUILD_NUMBER""
+        export TAGS="$TRAVIS_PULL_REQUEST_BRANCH-$TRAVIS_BUILD_NUMBER"
```

Comment:
you need to define `TRAVIS_PULL_REQUEST_BRANCH` probably to `github.head_ref`

Reply:
travis pull request branch had a different meaning that the correspective in github syntax,so i've created the action ".github/actions/travis-pull-request-branch" that translate the variable and to be sure to have the same behaviour through all the ci scripts. 
keep in mind that i prefered to not touch travis related stuff at this point to avoid breaking anything, but i've created different tasks on the migration epic in order to remove travis legacy code/env/scripts as soon the github action is live and the docker images are validated.

Reply:
ok, I was expecting to find `TRAVIS_PULL_REQUEST_BRANCH` via ctrl+f

Reply:
I don't see `TRAVIS_PULL_REQUEST_BRANCH` handled in travis-pull-request-branch (only `TRAVIS_EVENT_TYPE`)